HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4815,24 +4815,28 @@ def self.calculate_heat_pump_defrost_load_power_watts(heat_pump, unit_multiplier
     power_design = fan_watts_per_cfm * design_airflow_1x
     p_dot_blower = power_design * defrost_power_fraction
     # Based on manufacturer data for ~70 systems ranging from 1.5 to 5 tons with varying efficiency levels
-    p_dot_odu_fan = 44.348 * UnitConversions.convert(nominal_cooling_capacity_1x, 'Btu/hr', 'ton') + 62.452
-    rated_clg_cop = heat_pump.additional_properties.cool_rated_cops[-1]
+    if nominal_cooling_capacity_1x > 18000.0 # > 1.5 tons
+      p_dot_odu_fan = 44.348 * UnitConversions.convert(nominal_cooling_capacity_1x, 'Btu/hr', 'ton') + 62.452
+    else # < 1.5 tons, scale fan power to avoid negative p_dot_defrost
+      p_dot_odu_fan = 128.974 * (nominal_cooling_capacity_1x / 18000.0)
```

Comment:
@jonwinkler The actual change is here, I scaled down the fan power when cooling capacity is smaller than 1.5 tons, which is the smallest data point for regression.

Reply:
Thanks, @yzhou601. I think this looks reasonable. It took me a second to figure out how the constant `128.974` was determined, but I see now. Maybe add a comment to the code mentioning how that was calculated.

Reply:
Comment added, this PR will be ready for review after ci runs green